ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-commands.sh

```diff
@@ -133,6 +133,17 @@ function post-OCP-21588(){
     return $ret
 }
 
+function post-OCP-53907(){
+    echo "Test Start: ${FUNCNAME[0]}"
+    local arch 
+    arch=$(oc get clusterversion version -ojson|jq -r '.status.conditions[]|select(.type == "ReleaseAccepted")|.message')
+    if [[ "${arch}" != *"Multi"* ]]; then
```

Comment:
```suggestion
    if [[ "${arch}" != *architecture="Multi"* ]]; then
```
To be more accurate?

Reply:
btw, can we also check `ReleaseAccepted`'s status to be True for one more guard.

Reply:
Sounds like no need. ReleaseAccepted's status means no failures blocks an upgrade happen(for example, image verification or precondition check,etc). It's a post check about arch after upgrade succeed, which ensured in toimage step.

Reply:
updated the 1st one in [d585db5](https://github.com/openshift/release/pull/49743/commits/d585db51ead597b2083f1ac79b5a4b22346d47cf)